sys-auth/seatd/seatd-0.3.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Minimal seat management daemon and universal library"
+HOMEPAGE="https://git.sr.ht/~kennylevinsen/seatd"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://git.sr.ht/~kennylevinsen/seatd"
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://git.sr.ht/~kennylevinsen/seatd/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+LICENSE="MIT"
+SLOT="0/1"
+IUSE="examples logind"
+
+DEPEND="logind? ( || ( sys-auth/elogind sys-apps/systemd ) )"
```

Comment:
I think that it'll be nice to use `systemd`/`elogind` use flags instead of custom `logind` flag? You may also want to add `man` use flag to make `scdoc` an optional dependency the way it's done in sway ebuild?

Reply:
Will do for the `systemd`/`elogind` part, for some reason I thought they weren't globals and it turns out seatd needs to be recompiled if the `systemd`/`elogind` target changes.

As for the `man` for `scdoc` I don't really agree there as `scdoc` is really small (~70K on my machines) and doesn't depends on anything more than POSIX/C99, I think this should only be there for gigantic festivals like pandoc, docbook, …